quarkchain/evm/messages.py

```diff
@@ -559,10 +559,6 @@ def _apply_msg(ext, msg, code):
             # TODO: Why return success if the transfer failed?
             return 1, msg.gas, []
 
-    if msg.transfer_token_id != ext.default_state_token:
```

Comment:
just a reminder, should be careful here while #660 will address the potential problem